### PR TITLE
update korean translation

### PR DESCRIPTION
**Pull request by KimJeongSun**
## What are the changes the user will see?

update korean translations.

## What are the changes from a developer perspective?

Most of the content is based on [poke-corpus](https://abcboy101.github.io/poke-corpus/), and for the following items for which I obtained in-game screenshots, I have added line breaks to match those screenshots.
For the pokerogue-specific messages, if you have any better suggestions, please let me know.

### Screenshots/Videos

62355a094a638f709bb3a21d16b40178
FUc9PAgUsAEd1YN

Screenshot 2024-09-11 at 11 57 15 PM

## Checklist
- [ ] **I'm using `beta` as my base branch**
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [ ] Have I considered writing automated tests for the issue?
- [ ] If I have text, did I make it translatable and add a key in the English locale file(s)?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- [ ] Are the changes visual?
  - [ ] Have I provided screenshots/videos of the changes?

### CODE REVIEW

**Comment by KimJeongSun** [comment]
@EnochG1 @sodaMelon @returntoice plz review this! 🙇

File: src/locales/ko/battle.json [hunk 1]
```diff
@@ -62,6 +65,7 @@
   "skipItemQuestion": "아이템을 받지 않고 넘어가시겠습니까?",
   "itemStackFull": "{{fullItemName}}의 소지 한도에 도달했습니다.\n{{itemname}}[[를]] 대신 받습니다.",
   "eggHatching": "어라…?",
+  "eggSkipPrompt": "알스킵 하시겠습니까?",
```
Comment (sodaMelon) [inline src/locales/ko/battle.json]: This expression seems a bit awkward.

**Review by sodaMelon — COMMENTED** [review]
Everything is fine except for one.
i left comment about it.

File: src/locales/ko/battle.json (hunk 1 above)
Comment (EnochG1) [inline src/locales/ko/battle.json]:
```suggestion
  "eggSkipPrompt": "알 부화 요약 화면으로 넘어가시겠습니까?",
```

**Review by EnochG1 — COMMENTED** [review]
Please suggest your opinions.
@KimJeongSun @sodaMelon @returntoice

File: src/locales/ko/battle.json (hunk 1 above)
Comment (KimJeongSun) [reply src/locales/ko/battle.json]: 이 `정보`라는 단어가 좀 추상적으로 느껴지는데 ( egg summary 랑 stat summary 단어는 같지만 summary 의 의미 자체는 다를수있어서 `정보`라는 같은 단어로 번역해야하는지 확신이 없네요 )
맥락상 `정보` = `요약화면` 인걸로 보이는데 사실 `정보`라고 하면 너무 중의적으로 보이기도 하니까요.

`요약`은 어떤가요? ( `요약 화면` 도 괜찮지만 너무 길기도 하고 )
알 부화 요약으로 스킵하시겠습니까? (스킵이라고 하면 의미의 중복으로 느껴지기고 하네요) 
알 부화 요약으로 넘어가시겠습니까?

Comment (EnochG1) [reply src/locales/ko/battle.json]: 수정했습니다.

**Review by EnochG1 — APPROVED** [review]

File: src/locales/ko/battle.json [hunk 2]
```diff
@@ -62,6 +65,7 @@
   "skipItemQuestion": "아이템을 받지 않고 넘어가시겠습니까?",
   "itemStackFull": "{{fullItemName}}의 소지 한도에 도달했습니다.\n{{itemname}}[[를]] 대신 받습니다.",
   "eggHatching": "어라…?",
+  "eggSkipPrompt": "알 부화 요약 화면으로 넘어가시겠습니까?",
```
Comment (KimJeongSun) [inline src/locales/ko/battle.json]:
```suggestion
  "eggSkipPrompt": "알 부화 요약 화면으로 바로 넘어가시겠습니까?",
```

**Review by returntoice — APPROVED** [review]
LGTM

**Review by sodaMelon — APPROVED** [review]

**Comment by KimJeongSun** [comment]
base branch already merged to beta, so I open new PR with same updates https://github.com/pagefaultgames/pokerogue/pull/4196

**Pull request closed without merging**